src/perror.go

```diff
@@ -393,11 +396,11 @@ var MysqlErrNameToNum = map[string]int{
 	"OBSOLETE_ER_FLUSH_MASTER_BINLOG_CLOSED": 1186,
 	"ErrIndexRebuild": 1187,
 	"OBSOLETE_ER_INDEX_REBUILD": 1187,
-	"ER_MASTER": 1188,
+	"ER_SOURCE": 1188,
 	"ErrMaster": 1188,
```

Comment:
Should we create an issue in TiDB repo to clean up all these (unused?) error codes?

Reply:
You mean in pingcap/tidb? I don't think they are used there.  I only found one in `tests/integrationtest/t/privilege/privileges.test`: `--error ER_TABLEACCESS_DENIED_ERROR`, which should be fine.

Reply:
I guess the `ErrMaster` comes from pingcap/tidb, and I would assume most of the similar errors mentioning `Master` or `Slave` are not in use in TiDB, so they could be removed from pingcap/tidb I guess?

Reply:
We could update those. But maybe we should then also try to replace master/slave in other places as well like `dm-master`.